doc/manual/manual.tex

```diff
@@ -8168,7 +8168,53 @@ \subsubsection{Running benchmarks that require code}
 \texttt{solcx.prm} refers to the plugin as \texttt{./libsolcx.so}, i.e., in
 the current directory.
 
+\subsubsection{Onset of convection benchmark}
+\label{sec:benchmark-onset-of-convection}
+\textit{This section contributed by Max Rudolph, based on a course assignment for ``Geodynamic Modeling'' at Portland State University.}
 
+Here we use ASPECT to numerically reproduce numerically the results of a linear stability analysis for the onset of convection in a fluid layer heated from below. This exercise was assigned to students at Portland State University as a first step towards setting up a nominally Earth-like mantle convection model. Hence, representative length scales and transport properties for Earth are used. This cookbook consists of a jupyter notebook (\texttt{benchmarks/onset-of-convection/onset-of-convection.ipynb}) that is used to run aspect and analyze the results of several calculations. To use this code, you must compile aspect and give the path to the executable in the notebook as \texttt{aspect\_bin}. 
+
+The linear stability analysis for the onset of convection appears in Turcotte and Schubert \cite{TS14} (section 6.19). The linear stability analysis assumes the Boussinesq approximation and makes predictions for the growth rate (vertical velocity) of instabilities and the critical Rayleigh number $Ra_{cr}$ above which convection will occur. $Ra_{cr}$ depends only on the dimensionless wavelength of the perturbation, which is assumed to be equal to width of the domain. The domain has height $b$ and width $\lambda$ and the perturbation is described by:
+
+\begin{align*}
+T'(x,y) = T_0'\cos\left(\frac{2\pi x}{\lambda}\right)\sin\left(\frac{\pi y}{b} \right)
+\end{align}
+
+Note that because we place the bottom boundary of the domain at $y=0$ and the top at $y=b$, the perturbation vanishes at the top and bottom boundaries. This departs slightly from the setup in \cite{TS14} where the top and bottom boundaries of the domain are at $y=\pm b/2$.
+
+The analytic expression for the critical Rayleigh number, $Ra_{cr}$ is given in Turcotte and Schubert \cite{TS14} equation 6.319:
```

Comment:
comma after `\cite{...}`.

In the book, are formulas parenthesized? If so, add the parentheses here as well.